test/Garnet.test/GarnetServerConfigTests.cs

```diff
@@ -188,41 +188,96 @@ public void ImportExportRedisConfigLocal()
         [Test]
         public void ImportExportConfigAzure()
         {
+            if (!TestUtils.IsRunningAzureTests)
+            {
+                Assert.Ignore("Azure tests are disabled.");
+            }
+
             var AzureTestDirectory = $"{TestContext.CurrentContext.Test.MethodName.ToLowerInvariant()}";
             var configPath = $"{AzureTestDirectory}/test1.config";
             var AzureEmulatedStorageString = "UseDevelopmentStorage=true;";
```

Comment:
Add a unit test that uses the new version of identity.

Reply:
Any progress on this? We would need it to be confident that the feature works.

Reply:
I'm hoping to get to this by the end of this week

Reply:
Ping on this, thank you!

Reply:
Any update on this? Feel free to close and re-open when you have made progress on it, thanks.

Reply:
@badrishc I'm not sure how to test this locally, I got garnet to run but I'm not sure how to force garnet to save into the blob storage

Reply:
OK, I got garnet to interact with the azure blob storage I created.
![image](https://github.com/user-attachments/assets/c3ffdff5-989a-4370-951d-d987b62b91c1)